integration_tests/test.sh

```diff
@@ -28,14 +30,43 @@ for mod in $(ls); do
 done
 popd
 
+status=0
+failures=""
 echo "Doing schema validation..."
 for protocol in $(ls $ZGRAB_OUTPUT); do
     for outfile in $(ls $ZGRAB_OUTPUT/$protocol); do
         target="$ZGRAB_OUTPUT/$protocol/$outfile"
         echo "Validating $target [{("
         cat $target
         echo ")}]:"
-        python -m zschema validate schemas/__init__.py:zgrab2 $target
-        echo "validation of $target succeeded."
+        if ! python -m zschema validate schemas/__init__.py:zgrab2 $target; then
+            echo "Schema validation failed for $protocol/$outfile"
+            err="schema failure@$protocol/$outfile"
+            if [[ $status -eq 0 ]]; then
+                failures="$err"
+            else
+                failures="$failures, $err"
+            fi
+            status=1
+        else
+            echo "validation of $target succeeded."
+            scan_status=$($ZGRAB_ROOT/jp -u data.${protocol}.status < $target)
```

Comment:
Previously, failing mysql scans were caught because it was checking the server version in its own tests (and that was null in the case of a failed scan) -- but the SSH/postgres scans simply checked that the zgrab executable exited with status 0, and that the output was schematically valid -- and since e.g. `"status": "connection-refused"` is indeed valid according to the schema, they virtually never failed.